airflow/executors/kubernetes_executor.py

```diff
@@ -96,6 +96,8 @@ def __init__(self):  # pylint: disable=too-many-statements
         self.kube_labels = configuration_dict.get('kubernetes_labels', {})
         self.delete_worker_pods = conf.getboolean(
             self.kubernetes_section, 'delete_worker_pods')
+        self.delete_on_success = conf.getboolean(
```

Comment:
```suggestion
        self. delete_worker_pods_on_success = conf.getboolean(
```

I would suggest to use the explicit name as in future we can have delete namespace etc. WDYT @dimberman ?